v1_21_R2/src/main/java/net/citizensnpcs/nms/v1_21_R2/entity/EntityHumanNPC.java

```diff
@@ -341,6 +345,33 @@ public void remove(RemovalReason reason) {
         getAdvancements().save();
     }
 
+    @Override
+    public void setPos(double d0, double d1, double d2) {
+        if (npc != null) {
+            if (NPCMoveEvent.getHandlerList().getRegisteredListeners().length > 0) {
```

Comment:
Use &&

Reply:
fine. that is caused by coding habit as I want to make the condition chain clear and then we could inject further changes without separate out conditions again.
Will do it if really necessary.